### ISSUE TITLE
Firefox crash at startup

### ISSUE THREAD

**Issue by mbaker3** [sshnaidm/docker-ilo-client#1]
I've been trying to get this image to work and it seems there's an issue with Firefox. I get the error below when the container first starts up.

![image](https://user-images.githubusercontent.com/1426795/151881733-5dfe6c21-5c1d-4e0f-a98e-84d2f4b13f2a.png)

This is while attempting to connect to:
 - DL160 G6
 - iLO-100 (v4.26)

**Comment by sshnaidm** [comment]
Please try the newest version, updated repo and container

**Comment by mbaker3** [comment]
The updated image no longer shows an error when I VNC into the container but there is no UI. Just a black screen 
(same as screenshot above but no error dialog)

**Comment by sshnaidm** [comment]
Yeah, I could run firefox inside, but it kept crashing. Hard to know why because no logs..
`docker run --rm -it  -p 5900:5900 sshnaidm/docker-ilo-client`

**Comment by mbaker3** [comment]
I don't see anything that stands out either. Just in case it helps here are my logs.

[Log at 2022-02-15 8-56-51 PM.txt](https://github.com/sshnaidm/docker-ilo-client/files/8076097/Log.at.2022-02-15.8-56-51.PM.txt)

**Comment by mbaker3** [comment]
Oh, something to add. If I connect fast enough I can see Firefox launching before it quits. hmmm

**Comment by mbaker3** [comment]
I entered the container and installed `xinit`. Now I can reliably see Firefox before it crashes with some actual errors in the logs :). See attached.

I'm out of my depth on a setup like this but will pick away at this as I have time.
[Logs-withXinit.txt](https://github.com/sshnaidm/docker-ilo-client/files/8076220/Logs-withXinit.txt)

**Comment by divinity76** [comment]
fwiw i had the same problem, black screen or vnc client couldn't connect, with a HP iLO3 thing on a HP ProLiant DL580 G7

had more success with this project https://github.com/niclan/Javafox
